### PR TITLE
feat(Resource Controller): add "onetime_credentials" property in ResourceInstance and ResourceKey schemas

### PR DESCRIPTION
**Pull request by arshabbir**
## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [ ] No

### CODE REVIEW

File: resourcecontrollerv2/resource_controller_v2_examples_test.go [hunk 1]
```diff
@@ -77,7 +77,8 @@ var _ = Describe(`ResourceControllerV2 Examples Tests`, func() {
 		keyName                    string = "RcSdkKey1Go"
 		keyUpdateName              string = "RcSdkKeyUpdate1Go"
 		targetRegion               string = "global"
-		resourceGroupID            string = "testResourceGroupID"
+		// Replacing the placeholder value to the resourceGroupID
+		resourceGroupID string = "192ea4f81d444a4cbfc051353b6f8d2e"
```
Comment (pyrooka) [inline resourcecontrollerv2/resource_controller_v2_examples_test.go:81]: Variables that are sensitive or can change between tests should not be defined here. If you want to use something like that, you need to define it as an environment variable and load it. See [THIS](https://github.com/IBM/platform-services-go-sdk/blob/main/resourcecontrollerv2/resource_controller_v2_examples_test.go#L104) example.
Furthermore I see that the description of the `resourceGroup` and `resourceGroupID` variables are identical, so cannot the first one be used for this purpose and the second one removed?

**Review by pyrooka — CHANGES_REQUESTED** [review]
Besides the requested change, please post the result of the integration and examples test run here. It's required because we need to verify that the SDK continues interacting with the service perfectly.

File: resourcecontrollerv2/resource_controller_v2_integration_test.go [hunk 2]
```diff
@@ -225,6 +225,7 @@ var _ = Describe("Resource Controller - Integration Tests", func() {
 			fmt.Fprintf(GinkgoWriter, "GetResourceInstance() result:\n%s\n", common.ToJSON(result))
 
 			Expect(*result.ID).To(Equal(testInstanceCRN))
+			Expect(result.OnetimeCredentials).ToNot(BeNil())
```
Comment (pyrooka) [inline resourcecontrollerv2/resource_controller_v2_integration_test.go:228]: `result.OnetimeCredentials` is a pointer so it must be dereferenced before checking its value.
```suggestion
			Expect(*result.OnetimeCredentials).ToNot(BeNil())
```

File: resourcecontrollerv2/resource_controller_v2_integration_test.go [hunk 3]
```diff
@@ -1007,6 +1008,7 @@ var _ = Describe("Resource Controller - Integration Tests", func() {
 			fmt.Fprintf(GinkgoWriter, "GetResourceKey() result:\n%s\n", common.ToJSON(result))
 
 			Expect(*result.ID).To(Equal(testInstanceKeyCRN))
+			Expect(result.OnetimeCredentials).ToNot(BeNil())
```
Comment (pyrooka) [inline resourcecontrollerv2/resource_controller_v2_integration_test.go:1011]: `result.OnetimeCredentials` is a pointer so please dereference it before checking its value.
```suggestion
			Expect(*result.OnetimeCredentials).ToNot(BeNil())
```

**Review by pyrooka — CHANGES_REQUESTED** [review]
I forgot to add these to my previous review. :)

File: resourcecontrollerv2/resource_controller_v2_examples_test.go (hunk 1 above)
Comment (arshabbir) [reply resourcecontrollerv2/resource_controller_v2_examples_test.go]: Reverted this change to the previous state for this variable resource_group_id

File: resourcecontrollerv2/resource_controller_v2_integration_test.go (hunk 2 above)
Comment (arshabbir) [reply resourcecontrollerv2/resource_controller_v2_integration_test.go]: Addressed

File: resourcecontrollerv2/resource_controller_v2_integration_test.go (hunk 3 above)
Comment (arshabbir) [reply resourcecontrollerv2/resource_controller_v2_integration_test.go]: Addressed. Please verify

**Comment by arshabbir** [comment]
[out_examples.txt](https://github.com/user-attachments/files/16121626/out_examples.txt)
[output_latest_integration.txt](https://github.com/user-attachments/files/16121627/output_latest_integration.txt)
[output_latest_UTs.txt](https://github.com/user-attachments/files/16121628/output_latest_UTs.txt)

**Review by pyrooka — APPROVED** [review]
Looks good, thanks for the changes! One more thing: please do not forget to sign the CLA.

**Pull request merged**